### ISSUE TITLE
MetroBus.searchStops func errors with certain results

### ISSUE THREAD

**Issue by rhwood** [emma-k-alexandra/WMATA.swift#15]
The WMATA result for `https://api.wmata.com/Bus.svc/json/jStops?Lat=38.898303&Lon=-77.028099&Radius=100` returns
```
{
  "Stops": [{
    "StopID": "1001051",
    "Name": "F ST NW + 12TH ST NW",
    "Lon": -77.027878,
    "Lat": 38.897441,
    "Routes": ["54", "54*1"]
  }]
}
```
but this result leads to an error "The operation couldn’t be completed. (WMATA.WMATAError error 1.)" using this package.

**Comment by emma-k-alexandra** [comment]
Yeah, noticed this while running tests as well. I think it's because the route `54*1` isn't defined in [Routes.swift](https://github.com/emma-k-alexandra/WMATA.swift/blob/master/Sources/WMATA/Bus/Routes.swift). Routes seem to change fairly often, unlike with Rail. I think it may be wise to implement `Route` as a `struct` in the style of `Stop`. Possibly with a [`RawRepresentable`](https://developer.apple.com/documentation/swift/rawrepresentable) conformance for the both of them to improve ergonomics.

I'm working on a fairly large update that includes breaking changes for release soon, so it might be worth rolling a conversion of `Route` into that.

**Comment by emma-k-alexandra** [comment]
Fixed in 9.0.0
